.github/workflows/ci-cvxpy.yml

```diff
@@ -63,5 +63,5 @@ jobs:
 
       - name: Test CVXPY
         run: |
-          python3 -m pip install --break-system-packages pytest hypothesis
+          python3 -m pip install --break-system-packages pytest hypothesis numpy==1.26.0
```

Comment:
Yup, that'll do it, nearly there I see!

I'd make this [a matrix item](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow) so you test the project twice; once with the old numpy version and once with the current. At the top of the `ci-cvxpy.yml` file add another entry below `os`:

```yml
      matrix:
        os: [macos-latest]  # Add ubuntu-latest when cvxpy pytest issue is solved
        numpy: ['numpy==1.26.0', numpy]
```

then change the above line to

```console
          python3 -m pip install --break-system-packages pytest hypothesis ${{ matrix.numpy }}
```

and that should give you the whole workflow job twice; once with `numpy==1.26.0` and once with just `numpy`.

Reply:
Good idea, I'll do that. I'll change to requiring at least 1.26.0, which is a bit arbitrary, but should be fine.

Reply:
You forgot to update the `python3 -m pip install` line though, so it'll run the test twice but both times with numpy 1.26.0! :-D

Reply:
I've created a small follow-up PR, #213, to correct this.